### ISSUE TITLE
How to use scalikejdbc.PlayDBApiAdapterModule?  onfiguration error[Cannot initialize to database [scalikejdbc]]

### ISSUE THREAD

**Issue by StephanSchmidt** [scalikejdbc/scalikejdbc-play-support#52]
As suggested in another issue I've moved to PlayDBApiAdapterModule because I want to use HikariCP.

Sadly there is no readme for scalikejdbc.PlayDBApiAdapterModule and I've looked at the tests.

When I use 'default' in my configuration I get

    Configuration error[Cannot initialize to database [scalikejdbc]]

When I use 'scalikejdbc' as the name in my configuration I get

    Execution exception[[IllegalStateException: Connection pool is not yet initialized.(name:'default)]]

**Comment by StephanSchmidt** [comment]
Solved, somehow :-)

Looks like the problem was I had:

    scalikejdbc.global {
      loggingSQLErrors = true
      loggingSQLAndTime {
     [....]

How would I set those now?

**Comment by seratch** [comment]
You no longer need scalikejdbc.PlayModule. Just remove it from your conf file.

**Comment by StephanSchmidt** [comment]
Yes removed that, but I had 'scalikejdbc.global { .... ' in the config which caused

    Configuration error[Cannot initialize to database [scalikejdbc]]

How would I now use 'loggingSQLErrors = true' etc with PlayDBApiAdapterModule?

**Comment by seratch** [comment]
?

I guess your situation never happens unless you have the `scalikejdbc.global` under `db` namespace

**Comment by StephanSchmidt** [comment]
Ah. Thanks!

**Comment by seratch** [comment]
Good to hear :) Could you close this issue?
